squashfs-tools/squashfs-tools.json

```diff
@@ -2,7 +2,7 @@
     "name": "squashfs-tools",
     "buildsystem": "simple",
     "build-commands": [
-        "XZ_SUPPORT=1 LZO_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin INSTALL_MANPAGES_DIR=${FLATPAK_DEST}/man/man1"
+        "XZ_SUPPORT=1 LZO_SUPPORT=1 ZSTD_SUPPORT=1 GZIP_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin INSTALL_MANPAGES_DIR=${FLATPAK_DEST}/man/man1"
```

Comment:
Not related to your PR but this should not use `simple`,  `make-args` and `make-install-args` exist with `no-configure`.